src/test/java/com/anusha/testing/AppTest.java

```diff
@@ -42,4 +51,265 @@ public void shouldMultiplyTwoIntegers(){
         Assert.assertEquals(new Integer(8),result);
 
     }
+    @Test
+    public void shouldAddTwoStrings(){
+        String a = "Hello";
+        String b = "World";
+
+        String c = a+b;
+
+      //  Assert.assertEquals("HelloWorld",c);
+
+        assertThat(c, is("HelloWorld"));
+
+    }
+
+    // to check for printing even and odd numbers
+@Test
+    public void shouldREturnEvenValues(){
+    int[] myArray = {1, 20, 31, 9, 28, 37, 12, 45, 89, 96, 67};
+    List<Integer> evenList = ArrayUtils.getEvenNumbers(myArray);
+    boolean actual = evenList.contains(1);
+
+    //List<Integer> oddList = ArrayUtils.getOddNumbers(myArray);
+    //List<Integer> expected = Arrays.asList(20,28,12,96);
+
+    Assert.assertFalse(actual);
+   /* Collections.sort(evenList);
+    for (int even : evenList)
+        System.out.print(even + ","); */
+}
+
+@Test
+    public void shouldREturnOddValues(){
+    int[] myArray = {1, 20, 31, 9, 28, 37, 12, 45, 89, 96, 67};
+    List<Integer> oddList = ArrayUtils.getOddNumbers(myArray);
+    boolean actual = oddList.contains(20);
+
+    //
+    //List<Integer> expected = Arrays.asList(20,28,12,96);
+
+    Assert.assertFalse(actual);
+    Collections.sort(oddList);
+    for (int odd : oddList)
+        System.out.print(odd + ",");
+}
+
+// to check for the arraylist size
+@Test
+    public void shouldDisplayArrayListSize(){
+    ArrayList<Integer> myArr = new ArrayList<>();
+    myArr.add(15);
+    myArr.add(98);
+    myArr.add(75);
+    myArr.add(59);
+    myArr.add(88);
+    myArr.add(9);
+    myArr.add(25);
+
+    Assert.assertEquals(7,myArr.size());
+}
+
+//to check whether first element is removed from the arraylist
+@Test
+ public void shouldDisplayRemovedFirstElementFromArrayList() {
+
+    ArrayList<Integer> myArr = new ArrayList<>();
+    myArr.add(15);
+    myArr.add(98);
+    myArr.add(75);
+    myArr.add(59);
+    myArr.add(88);
+    myArr.add(9);
+    myArr.add(25);
+
+    //1. method
+  /* Assert.assertEquals(7,myArr.size());
+   myArr.remove(0);
+   Assert.assertEquals(6,myArr.size()); */
+
+    //2.method
+    Assert.assertTrue(myArr.contains(15));
+    myArr.remove(0);
+    Assert.assertFalse(myArr.contains(15));
+}
+
+    //to check whether last element is removed from the arraylist
+@Test
+    public void shouldDisplayRemovedLastElementFromArrayList(){
+    ArrayList<Integer> myArr = new ArrayList<>();
+    myArr.add(15);
```

Comment:
can you look at other ways of initialising test data ? 

May be explore use of Arrays.asList method